src/test/resources/bundles/messageOrQuestionExample/config.json

```diff
@@ -49,7 +49,7 @@
     "confirmationState": {
       "name": "Confirmation",
       "description": "Please confirm",
-      "showDetailCardHeader": false,
+      "showDetailCardHeader": true,
```

Comment:
Same remark than previously. 
I propose to have only one 
"showDetailCardHeader" : true
in opfab, for automatic tests, to be sure there is no regression in the future.